### PR TITLE
Add Teachable Week names

### PR DESCRIPTION
**Pull request by oliverochman**

### CODE REVIEW

File: weeks/week_1.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ Första veckans utmaning handlar om att bygga en enkel Bankomatsimulator. Låter
 **Test driven development and introduction to Ruby**
 The first week's challenge is to build a simple ATM Simulator. The challenge is about understanding how an everyday situation might look like when you think about it in terms of code and how different objects interact with each other. During the week, we introduce a number of new terms and concepts that lay the foundation for your future development. We practice test driven development and we also struggle with version control and more advanced Ruby syntax. During this week we also introduce a user-focused approach to program development. The week ends with your first individual Weekend Challenge.
 
+### Teachable week name
```
Comment (tochman) [inline weeks/week_1.md]: Teachable Course:

File: weeks/week_1.md [hunk 2]
```diff
@@ -10,6 +10,9 @@ Första veckans utmaning handlar om att bygga en enkel Bankomatsimulator. Låter
...
+Object Oriented Programming Fundamentals
```
Comment (tochman) [inline weeks/week_1.md]: Kanske en länk?

File: weeks/week_1.md (hunk 1 above)
Comment (oliverochman) [reply weeks/week_1.md]: fixed

File: weeks/week_1.md (hunk 2 above)
Comment (oliverochman) [reply weeks/week_1.md]: fixed

**Pull request merged**